src/main/resources/entity-types/circulation/simple_loan_policy.json5

```diff
@@ -273,6 +262,32 @@
       queryable: true,
       visibleByDefault: false,
       valueGetter: ":sourceAlias.jsonb->'loansPolicy'->>'closedLibraryDueDateManagementId'",
+      values: [
+        {
+          label: 'CURRENT_DUE_DATE',
+          value: 'CURRENT_DUE_DATE',
+        },
+        {
+          label: 'END_OF_THE_PREVIOUS_OPEN_DAY',
+          value: 'END_OF_THE_PREVIOUS_OPEN_DAY',
+        },
+        {
+          label: 'END_OF_THE_NEXT_OPEN_DAY',
+          value: 'END_OF_THE_NEXT_OPEN_DAY',
+        },
+        {
+          label: 'CURRENT_DUE_DATE_TIME',
+          value: 'CURRENT_DUE_DATE_TIME',
+        },
+        {
+          label: 'END_OF_THE_CURRENT_SERVICE_POINT_HOURS',
+          value: 'END_OF_THE_CURRENT_SERVICE_POINT_HOURS',
+        },
+        {
+          label: 'BEGINNING_OF_THE_NEXT_OPEN_SERVICE_POINT_HOURS',
+          value: 'BEGINNING_OF_THE_NEXT_OPEN_SERVICE_POINT_HOURS',
+        },
+      ]
```

Comment:
This list hasn't changed in 5 years (https://github.com/folio-org/mod-circulation/blob/master/src/main/java/org/folio/circulation/domain/policy/DueDateManagement.java), so I figured that it's safe to hard-code here

Reply:
you're giving me mod-calendar flashbacks...